### PR TITLE
fix: add qa property for some components

### PR DESCRIPTION
**Pull request by KirillDyachkovskiy**

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -78,7 +80,7 @@ export const Card = React.forwardRef<HTMLDivElement, CardProps>(function Card(pr
         <div
             style={style}
             ref={ref}
-            role={isClickable ? 'button' : undefined}
+            role={isClickable ? 'button' : role}
```
Comment (korvin89) [inline src/components/Card/Card.tsx:83]: It seems a little bit unfair if user sets property `role` and it does not work in some cases

Comment (amje) [reply src/components/Card/Card.tsx:83]: What is the purpose of this overall?

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Text/Text.tsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
+import type {QAProps} from '../types';
+
 import {colorText} from './colorText/colorText';
 import type {ColorTextBaseProps} from './colorText/colorText';
 import {text} from './text/text';
 import type {TextBaseProps} from './text/text';
 
-export interface TextProps extends TextBaseProps, ColorTextBaseProps {
+export interface TextProps extends TextBaseProps, ColorTextBaseProps, QAProps {
```
Comment (amje) [inline src/components/Text/Text.tsx:10]: qa prop is not used

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Text/Text.tsx (hunk 2 above)
Comment (KirillDyachkovskiy) [reply src/components/Text/Text.tsx:10]: it spreads out from `rest`

**Review by amje — APPROVED** [review]

**Review by Lunory — APPROVED** [review]

**Review by IsaevAlexandr — APPROVED** [review]

**Review by korvin89 — APPROVED** [review]

**Review by jhoncool — APPROVED** [review]

File: src/components/Tooltip/README.md [hunk 3]
```diff
@@ -25,4 +25,5 @@ import {Tooltip} from '@gravity-ui/uikit';
 | content    | Content that will be shown in the `Tooltip`                                             |                `React.ReactNode`                 |         |
 | closeDelay | Number of ms to delay hiding the `Tooltip` after the hover ends                         |                     `number`                     |   `0`   |
 | openDelay  | Number of ms to delay showing the `Tooltip` after the hover begins                      |                     `number`                     |  `250`  |
-| placement  | `Tooltip` position relative to its anchor                                               | [`PopupPlacement`](../Popup/README.md#placement) |         |
+| placement  | `Tooltcip` position relative to its anchor                                              | [`PopupPlacement`](../Popup/README.md#placement) |         |
```
Comment (ogonkov) [inline src/components/Tooltip/README.md:28]: typo here -- Toolt**c**ip

File: src/components/Tooltip/Tooltip.tsx [hunk 4]
```diff
@@ -47,6 +47,7 @@ export const Tooltip = (props: TooltipProps) => {
                 disableEscapeKeyDown
                 disableOutsideClick
                 disableLayer
+                qa={qa}
```
Comment (ogonkov) [inline src/components/Tooltip/Tooltip.tsx:50]: Why not select component by role tooltip?

**Review by goshander — APPROVED** [review]

**Review by SeqviriouM — APPROVED** [review]

**Pull request merged**